### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jab0707**
added template readme stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# UncertainSCI
\ No newline at end of file
+#UncertainSCI
+> Short blurb about what your product does.
+A python based toolkit to harness modern techniques to estimate statistical uncertainty in biomedical simulations by integrating these techniques with well-estabilsied biomedical simulation software.
```
Comment (dcwhite) [inline README.md:3]:
```suggestion
A python based toolkit to harness modern techniques to estimate statistical uncertainty in biomedical simulations by integrating these techniques with well-established biomedical simulation software.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+## Release History
+
+* 0.1.1
+
+
+## Meta
+
+
+
+Distributed under the XYZ license. See ``LICENSE`` for more information.
```
Comment (akilnarayan) [inline README.md:50]: Maybe MIT license like SCIRun? Throughts?

Comment (jab0707) [reply README.md:50]: MIT seems most appropriate

**Comment by jab0707** [comment]
Adding Basic Readme Structure

**Pull request merged**